src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/DateTimePickerTests.cs

```diff
@@ -271,6 +284,178 @@ public void DateTimePicker_CalendarMonthBackground_SetInvalid_ThrowsArgumentExce
         act.Should().Throw<ArgumentException>();
     }
 
+    [WinFormsFact]
+    public void DateTimePicker_CalendarFont_GetSet_ReturnsExpected()
+    {
+        Font expectedFont = new("Arial", 8.25f);
+
+        _dateTimePicker.CalendarFont = expectedFont;
+        _dateTimePicker.CalendarFont.Should().Be(expectedFont);
+
+        _dateTimePicker.CalendarFont = null;
+        _dateTimePicker.CalendarFont.Should().Be(_dateTimePicker.Font);
+    }
+
+    [WinFormsFact]
+    public void DateTimePicker_Checked_GetSet_ReturnsExpected()
+    {
+        _dateTimePicker.Checked.Should().BeTrue();
+
+        _dateTimePicker.Checked = false;
+        _dateTimePicker.Checked.Should().BeFalse();
+
+        _dateTimePicker.Checked = true;
+        _dateTimePicker.Checked.Should().BeTrue();
+    }
+
+    [WinFormsFact]
+    public void DateTimePicker_CustomFormat_GetSet_ReturnsExpected()
+    {
+        _dateTimePicker.CustomFormat.Should().BeNull();
+
+        _dateTimePicker.Value = new DateTime(2021, 12, 31);
+        _dateTimePicker.CreateControl();
+
+        _dateTimePicker.CustomFormat = "yyyy/MM/dd";
+        _dateTimePicker.Format = DateTimePickerFormat.Custom;
+        _dateTimePicker.Text.Should().Be("2021/12/31");
+
+        _dateTimePicker.CustomFormat = "MM/dd/yyyy";
+        _dateTimePicker.Text.Should().Be("12/31/2021");
+
+        _dateTimePicker.CustomFormat = null;
+        _dateTimePicker.CustomFormat.Should().BeNull();
+    }
+
+    [WinFormsFact]
+    public void DateTimePicker_DropDownAlign_GetSet_ReturnsExpected()
+    {
+        _dateTimePicker.DropDownAlign.Should().Be(LeftRightAlignment.Left);
+
+        _dateTimePicker.DropDownAlign = LeftRightAlignment.Right;
+        _dateTimePicker.DropDownAlign.Should().Be(LeftRightAlignment.Right);
+
+        _dateTimePicker.DropDownAlign = LeftRightAlignment.Left;
+        _dateTimePicker.DropDownAlign.Should().Be(LeftRightAlignment.Left);
+    }
+
+    [WinFormsFact]
+    public void DateTimePicker_MaxDate_GetSet_ReturnsExpected()
+    {
+        _dateTimePicker.MaxDate.ToString().Should().Be("12/31/9998 12:00:00 AM");
+
+        var expectedDate = new DateTime(2022, 12, 31);
```

Comment:
Please use target type new
```suggestion
        DateTime expectedDate = new(2022, 12, 31);
```